lambda/lambda-returns-usr-info.py

```diff
@@ -0,0 +1,80 @@
+import json
+import boto3
+from botocore.exceptions import ClientError
+import psycopg2
+from psycopg2 import OperationalError
+
+
+def getCredentials():
+    credential = {}
+    
+    secret_name = "mysecretname"
+    region_name = "us-east-1"
+    
+    client = boto3.client(
+      service_name='secretsmanager',
+      region_name=region_name
+    )
+    
+    try:
+        get_secret_value_response = client.get_secret_value(
+            SecretId=secret_name
+        )
+    except ClientError as e:
```

Comment:
this is good, but it's unneeded, as the ClientError error class will state what the problem is when the exception is raised.

Please replace all the if statements with one `raise e`